src/Components/FormContainer.tsx

```diff
@@ -13,14 +13,17 @@ export const FormContainer = () => {
   const [answers, setAnswer] = useState<string[]>([]);
 
   const [currentQuestionIndex, changeQuestion] = useState(0)
+  console.log(answers)
```

Comment:
I like the conditional logic & error handling here for only rendering the submit button on the last question